en/contribute/maintainers.md

```diff
@@ -0,0 +1,93 @@
+
+## Recruitment process
+
+If you would like to join the PX4 Maintainers Team, please do the following:
+
+1. Read the [role description](#dronecode-maintainer-role-description), and make sure you understand the responsibilities & your role.
+2. Contact the relevant maintainer listed in the [PX4-Autopilot README](https://github.com/PX4/PX4-Autopilot#maintenance-team) on [Discord server](support.md#forums-and-chat), or any of the preferred method.
+3. Express your interest in becoming a maintainer, and specify which sector you would like to maintain.
+4. Bring this up in the [weekly maintainers call](#maintainers-call), and if there's a general consensus on approval, you will become a maintainer!
+5. If approved, go through the [onboarding process](#onboarding-process)
+
+## Onboarding process
+
+After you become a maintainer, you will go through the following process:
+
+1. **Discord** server admin will grant you the `dev team` role, which gives you:
+   1. Permission to speak during maintainers call
+   2. Access to the `#maintainers` channel
+   3. Access to the user group / dronecode member companies channels
+2. **Github** PX4 Workspace admin will grant you the [`Dev Team`](https://github.com/orgs/PX4/teams/dev-team) role, which gives you:
+   1. Permission to merge the PR of any of PX4 workspace repositories after it's approved
+   2. Permission to trigger Github actions when a new contributor opens a PR
+   3. Permission to edit Issue/PR contents
+3. **Dronecode member** (Junwoo/Ramon) will:
+   1. Update the PX4 code repository [README](https://github.com/PX4/PX4-Autopilot/blob/main/README.md) to include your information
+   2. Update the PX4 website [Maintainers section](https://px4.io/community/maintainers/) to include your information
+   3. Add your information to the internal Dronecode database of maintainers to keep you in sync
+   4. Create a social media post about your new maintainers role after a short interview
+
+## Dronecode Maintainer Role Description
+
+### Summary
+
+Dronecode Maintainer will lead/manage the development of a **specific category (referred as category below)** of the PX4 ecosystem. With this role, you will be at the epicenter of developing the world’s top most popular flight control software system!
+
+### Responsibilities
+
+1. Take charge of overseeing the development in their category
+2. Provide guidance/advice on community members in their category
+3. Help with PR review and Issues in their category
+4. Coordinate with the maintainer group
+5. Attend the weekly [Maintainers meeting](#maintainers-call)
+6. Help create and maintain a Roadmap for PX4
+7. Adhere to Dronecode's [Code of Conduct](https://github.com/Dronecode/foundation/blob/main/CODE-OF-CONDUCT.md)
+
+### Qualifications
+
+1. Experience in the category field with proven track record of contributions
+2. Good overview of the category in PX4 ecosystem
+3. Approval from your employer
```

Comment:
As @Jaeyoung-Lim, since Maintainers role isn't paid, I don't think this is necessary. What do you think? (Everyone)

Reply:
I don't think it is necessary because this should not be taking up "work time".